net/portmapper/portmapper.go

```diff
@@ -876,6 +883,11 @@ var uPnPPacket = []byte("M-SEARCH * HTTP/1.1\r\n" +
 	"ST: ssdp:all\r\n" +
 	"MAN: \"ssdp:discover\"\r\n" +
 	"MX: 2\r\n\r\n")
+var uPnPIGDPacket = []byte("M-SEARCH * HTTP/1.1\r\n" +
+	"HOST: 239.255.255.250:1900\r\n" +
+	"ST: urn:schemas-upnp-org:device:InternetGatewayDevice:1\r\n" +
```

Comment:
It's not enough to send one UDP packet with both of these ST values or something?

Reply:
We don't know. We could try it, but we also don't know whether multiple ST fields would break other routers.

The SAGEMCOM model known to not respond well to `ssdp:all` hasn't had any for sale on eBay recently.